app/src/main/kotlin/com/wire/android/ui/home/conversationslist/common/ConversationItemFactory.kt

```diff
@@ -97,11 +98,19 @@ private fun GeneralConversationItem(
                     eventType = conversation.badgeEventType,
                     clickable = onConversationItemClick,
                     trailingIcon = {
-                        if (hasOnGoingCall)
-                            JoinButton(buttonClick = onJoinCallClick)
-                        else if (mutedStatus != MutedConversationStatus.AllAllowed) {
-                            MutedConversationBadge(onMutedIconClick)
-                            HorizontalSpace.x8()
+                        Row {
+                            if (hasOnGoingCall) {
+                                HorizontalSpace.x8()
+                                JoinButton(buttonClick = onJoinCallClick)
```

Comment:
`JoinButton` has some default end padding, I don't know why it doesn't also have matching start padding and where it's needed but that can interfere with these added spaces. Maybe it would  be better to add `PaddingValues` as an optional parameter to `JoinButton` and add spaces this way?

Reply:
Added paddings inside JoinButton